sonar-plugin-api/src/main/java/org/sonar/api/server/debt/internal/DefaultDebtRemediationFunction.java

```diff
@@ -62,29 +62,50 @@ public Type type() {
     return type;
   }
 
+  /**
+   * @deprecated since 5.5, replaced by {@link #gapMultiplier}
+   */
   @Override
   @CheckForNull
+  @Deprecated
   public String coefficient() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)